common/config/lib/src/remote_config_keys.dart

```diff
@@ -8,6 +8,21 @@ class RemoteConfigKeys {
   /// make airtime transfer request by sms.
   static const userCanRequestAirtimeBySms = 'users_can_request_airtime_by_sms';
 
+  /// Feature follow us enabled.
+  static const followUsEnabled = 'follow_us_enabled';
```

Comment:
I don't think we need the remote config to display the text `Follow us`.
```suggestion
```

Reply:
ok ... i delete it